types/package.json

```
@@ -1,13 +1,29 @@
 {
   "name": "@joystream/types",
-  "version": "0.20.3",
+  "version": "0.20.4",
   "description": "Types for Joystream Substrate Runtime - Carthage release",
-  "main": "index.js",
-  "types": "index.d.ts",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./lib/mjs/index.js",
+      "require": "./lib/cjs/index.js"
+    }
+  },
+  "typesVersions": {
+    "*": {
+      ".": [
+        "lib/index.d.ts"
+      ],
+      "primitives": [
+        "lib/primitives.d.ts"
+      ]
+    }
+  },
   "scripts": {
     "prepublishOnly": "yarn clean && yarn build",
     "prepack": "yarn clean && yarn build",
-    "compile": "tsc --build tsconfig.json",
+    "compile": "tsc --project tsconfig.esm.json & tsc --project tsconfig.cjs.json",
```

Better to fail if esm build fails, with single `&` it was running in background. I think what we want is:

```suggestion
    "compile": "tsc --project tsconfig.esm.json && tsc --project tsconfig.cjs.json",
```